docs/decisions/0003-migrating-to-react-virtuoso.md

```diff
@@ -0,0 +1,31 @@
+# Migrating Parsley Virtualization to react-virtuoso
+
+* status: proposed
+* date: 2022-02-22
```

Comment:
```suggestion
* date: 2023-02-22
```

Reply:
oops i'm still stuck in 2022 gonna take me a few months to adjust :/